content/en/Getting started/planning.md

```diff
@@ -10,8 +10,11 @@ description: >
 Now you can set a date for the pentest.
 {{% /pageinfo %}}
 
-By default, we can help you schedule pentests with a start date at least
-two (2) business days after you select "Submit for Review."
+<!-- For content, see https://github.com/cobalthq/cobalt-product-public-docs/blob/main/layouts/shortcodes/big-pic-UI-steps.md -->
```

Comment:
Until the PR is merged, you'll see this file in the following directory: https://github.com/cobalthq/cobalt-product-public-docs/tree/DOCS-70_incorporate_5steps_article_from_zendesk/layouts/shortcodes